src/strategies/ens-10k-club/index.ts

```diff
@@ -0,0 +1,125 @@
+import { subgraphRequest } from '../../utils';
+import { BigNumber } from '@ethersproject/bignumber';
+import { getAddress } from '@ethersproject/address';
+
+import { TOKEN_IDS_999_CLUB } from './999-club-token-ids';
+import { TOKEN_IDS_10K_CLUB } from './10k-club-token-ids';
```

Comment:
Hey, what are these ids? these should not be on this repo., maybe should be on an IPFS file and fetch it when required